### PR TITLE
Fix golint issues in pkg/kubelet/types

### PR DESCRIPTION
**Pull request by geegeea**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
This PR fixes golint issues in `pkg/kubelet/types`

**Which issue(s) this PR fixes**:
Part of #68026

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
```
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```

```

### CODE REVIEW

**Comment by ehashman** [comment]
/lgtm
/priority backlog

Mostly docstring updates, no real semantic changes

File: pkg/kubelet/types/pod_update.go [hunk 1]
```diff
@@ -34,34 +35,38 @@ const (
 // PodOperation defines what changes will be made on a pod configuration.
 type PodOperation int
 
+// These constants identify the PodOperations that can be made on a pod configuration.
 const (
-	// This is the current pod configuration
+	// SET is the current pod configuration.
```
Comment (SergeyKanzhelev) [inline pkg/kubelet/types/pod_update.go:40]: nit: make it consistent?

```suggestion
	// SET: is the current pod configuration.
```

File: pkg/kubelet/types/pod_update.go [hunk 2]
```diff
@@ -34,34 +35,38 @@ const (
...
 	SET PodOperation = iota
-	// Pods with the given ids are new to this source
+	// ADD: Pods with the given ids are new to this source.
 	ADD
-	// Pods with the given ids are gracefully deleted from this source
+	// DELETE: Pods with the given ids are gracefully deleted from this source.
```
Comment (SergeyKanzhelev) [inline pkg/kubelet/types/pod_update.go:44]: I don't really like how you approached it with adding semicolon. Would love to see more human readable thing. But it's fine.

**Comment by SergeyKanzhelev** [comment]
(for /hack):
/assign @dims

**Comment by SergeyKanzhelev** [comment]
@geegeea please rebase.

File: pkg/kubelet/types/pod_update.go (hunk 2 above)
Comment (geegeea) [reply pkg/kubelet/types/pod_update.go]: I've updated the comments to be more human readable. Please take a look.

**Comment by geegeea** [comment]
> Gurleen Grewal please squash commits.

@SergeyKanzhelev done rebase.

**Pull request merged**